docs/tutorials/typescript/nextjs/index.md

```diff
@@ -1,290 +1,626 @@
 ---
-id: nextjs-tutorial
+id: build-one-click-order-app-nextjs
+title: Build a one-click order application with TypeScript and Next.js
 sidebar_position: 2
 keywords: [TypeScript, temporal, sdk, tutorial, NextJS]
-tags: [TypeScript, SDK]
+tags:
+  - TypeScript
 last_update:
-  date: 2021-10-01
-title: Integrating Temporal into an existing Next.js application
-description: Explore how Temporal integrates into an existing Next.js application using Next.js API routes. This gives you the ability to write full-stack, long-running applications end to end in TypeScript.
+  date: 2024-06-25
+description: Build a One-Click Buy application with Next.js and integrate Temporal using Next.js API routes to create a durable order processing backend.
 image: /img/temporal-logo-twitter-card.png
+repository: https://github.com/temporalio/nextjs-temporal-one-click-template/
 ---
 
 ![Temporal TypeScript SDK](/img/sdk_banners/banner_typescript.png)
 
-import { OutdatedNotice } from '@site/src/components'
+### Introduction
 
-<OutdatedNotice />
+When you're building an e-commerce application, you want to give customers a great user experience.
+You also need to make sure that any calls to external services, like databases, payment gateways, and other tools, are reliable.
+You can deliver a great experience across the stack by integrating a Temporal Workflow with Next.js.
```

Comment:
If we are assuming no Temporal knowledge of the reader, I would suggest a quick sentence as to why Temporal will deliver a great experience like you mention. How about something like:

"You can deliver a great experience across the stack by integrating a Temporal Workflow with Next.js. Temporal provides fault tolerance and ensures that long-running processes and background tasks complete successfully, even in the event of failures, making it ideal for critical business operations."